licenses.d2iq.yaml

```diff
@@ -139,13 +139,13 @@ resources:
       - license_path: LICENSE
         ref: ${image_tag}
         url: https://github.com/istio/proxy
-  - container_image: docker.io/jaegertracing/all-in-one:1.46.0
+  - container_image: docker.io/jaegertracing/all-in-one:1.52.0
     sources:
       - license_path: LICENSE
         notice_path: NOTICE
         ref: v${image_tag}
         url: https://github.com/jaegertracing/jaeger
-  - container_image: docker.io/jaegertracing/jaeger-operator:1.46.0
+  - container_image: docker.io/jaegertracing/jaeger-operator:1.49.0
```

Comment:
noticed the discrepancy and opened a fix upstream to use the expected version https://github.com/jaegertracing/helm-charts/pull/536